.github/workflows/cherry-pick.yaml

```diff
@@ -7,15 +9,18 @@ on:
 jobs:
   cherry_pick_release_v0_28:
     runs-on: ubuntu-latest
-    name: Cherry pick into release-0.28 brach
+    name: Cherry-pick into release-0.28 brach
     if: ${{ contains(github.event.pull_request.labels.*.name, 'cherry-pick/release-0.28') && github.event.pull_request.merged == true }}
```

Comment:
Cant we rationalize it to use any branch (cherry-pick/*). This will make sure we can support cherry pick to any branch and as when starts for GH, we do not need to modify and also this can be used in future as well without changes.

Reply:
+1 to this scenario but we would have to create the label first I guess to support (cherry-pick/*)

Reply:
What about having a comment like `/cherry-pick <branch-name>` instead of label because we have to create the label every time.

Reply:
@rajaskakodkar problem with argument passing is there can be possibility of the branch passed not existing. Using labels we can to some extend/reasonably assume the target branch is available

Reply:
re: rationalise it use any branch
will revisit generalising this to work with different release branches

re: we would have to create the label first
yes (created only labels for which cherry-picks are open for now) 

re: /cherry-pick <branch-name>
think it can be done too, could've been better if the issue_comment activity is parsed elsewhere, for now kept it label based so only merge of the PR triggers an event

thanks for reviews! will revisit the suggestions in follow up.